Formula/opentsdb.rb

```diff
@@ -0,0 +1,121 @@
+require "socket"
+
+class Opentsdb < Formula
+  desc "Scalable, distributed Time Series Database."
+  homepage "http://opentsdb.net/"
+  url "https://github.com/OpenTSDB/opentsdb/releases/download/v2.2.0/opentsdb-2.2.0.tar.gz"
+  sha256 "5689d4d83ee21f1ce5892d064d6738bfa9fdef99f106f45d5c38eefb9476dfb5"
+
+  option "without-lzo", "Don't use LZO Compression"
+
+  if build.without? "lzo"
+    depends_on "hbase"
+  else
+    depends_on "hbase" => "with-lzo"
```

Comment:
We don't actually have a simple way to mandate you must have built with a default option at this point. You can _kind of_ force it with a Requirement [like this](https://github.com/Homebrew/homebrew-core/blob/ed890fc46b99f54e387caae92d7e5dd01bfdbbac/Formula/coq.rb#L1-L11) but it's a dance.